tools/ncf.py

```diff
@@ -147,7 +146,7 @@ def parse_bundlefile_metadata(content, bundle_type):
         else:
           res[tag] = match.group(1)
 
-    match = re.match("[^#]*bundle\s+agent\s+(\w+)(\(([^)]+)\))?.*$", line)
+    match = re.match("[^#]*bundle\s+agent\s+([0-9a-zA-Z_]+)(\(([^)]+)\))?.*$", line, flags=re.UNICODE)
```

Comment:
why not \w anymore ? is it to prévent accents in bundle names ? (this is legal in cfengine)